google/cloud/forseti/common/gcp_api/iam.py

```diff
@@ -284,10 +286,17 @@ def get_curated_roles(self, parent=None):
         try:
             paged_results = self.repository.roles.list(parent=parent,
                                                        view='FULL')
-            return api_helpers.flatten_list_results(paged_results, 'roles')
+            flattened_results = api_helpers.flatten_list_results(paged_results,
+                                                                 'roles')
+            LOGGER.debug('Getting information about organization roles,'
+                         ' parent = %s, flattened_results = %s',
+                         parent, flattened_results)
+            return flattened_results
         except (errors.HttpError, HttpLib2Error) as e:
-            LOGGER.warn(api_errors.ApiExecutionError(parent, e))
-            raise api_errors.ApiExecutionError('projects_roles', e)
+            raise api_errors.ApiExecutionError(
+                API_EXECUTION_ERROR_ARG_FORMAT.format('project_roles',
```

Comment:
It seems really weird to pull in a format constant from another module. Is there another way?

Reply:
it's now using the new constructor instead